website/content/en/blog/2025-09-navigating-platform-engineering

```diff
@@ -0,0 +1,59 @@
+---
+title:  "Navigating Platform Engineering with the maturity model assessment"
+slug: navigating-platform-engineering
+date:   2025-09-02 01:00:00 +0000
+author: Steve Fenton
+categories:
+- Article
+tags:
+- WG Platforms
+- Community Contributions
+---
+
+The [CNCF Platform Engineering community](https://cloudnativeplatforms.com) has been working on an assessment to accompany the [Platform Engineering Maturity Model](https://cloudnativeplatforms.com/whitepapers/platform-eng-maturity-model/). It's now available as a preview for feedback, but it's crucial to understand its purpose to know what it can do for you and what you need to put in once you get your score.
+
+Help us out by [trying out the assessment](https://cloud-native-platform-engineering.github.io/pemm-assessment/) and [giving us feedback](https://docs.google.com/forms/d/e/1FAIpQLScXru41BVVQDipxuSCQaNmo0GcBpBM8jHhDbX3pQskJXFgV8A/viewform).
+
+## Platform orienteering
+
+Picture yourself at the edge of a vast forest. You know there's a destination somewhere out there, but between you and your goal lies unfamiliar terrain filled with hills, valleys, streams, and dense woodland. This is orienteering, the sport of navigating unknown territory using only a map and compass.
+
+You can take part in orienteering with minimal kit. If you have a map, you can work out the rest using landmarks. Perhaps you'll spot that distinctive hill in the distance, follow the bend of a river, or use a church spire as a reference point. But it gets a whole lot easier if you have a compass to tell you which direction you're facing.
+
+That's why we wanted to add an assessment tool to accompany the Platform Engineering maturity model. Just like orienteering requires both a map and a compass working together, a successful Platform Engineering transformation needs a clear picture of the landscape and a way to determine your current position.
+
+## The maturity model is the map
+
+In orienteering, a map is your window into the landscape. It shows you the contour lines that reveal steep climbs and gentle slopes, the symbols that mark forests, clearings, and water features, and the network of paths that connect different areas. But here's what's crucial to understand: the map shows you the terrain as it exists, not where you are within that terrain.
+
+The Platform Engineering maturity model works the same way. It's a detailed map that describes the features of the Platform Engineering landscape. It has the organizational structures, the technical capabilities, the process maturity levels, and the cultural elements that define different stages of platform evolution. It shows you what "good" looks like at various levels of sophistication.
+
```

Comment:
We could add the maturity model image here, this is present in the PDF version of the maturity model document. 

<img width="655" height="467" alt="image" src="https://github.com/user-attachments/assets/0bd97739-0f73-4b3f-835d-c4044b651627" />